### ISSUE TITLE
Backup script ignores Environment::getContext

### ISSUE THREAD

**Issue by schrej-zz** [Apen/additional_scheduler#20]
I have 3 Typo3 installations (DEV/Test/Production).
The backup script reads the DB data (DB name, User, etc.) out of the file `LocalConfiguration.php` and ignores the definitions in `AdditionalConfiguration.php`.
Therefore no matter in which installation I am in, only the DB definded in `LocalConfiguration.php` is backuped.
Either
- the extension also hands over the Environment::getContext to the script and also analyzes `AdditionalConfiguration.php` (which seems more complex to me) or
- the neccessary DB data are read within the extension and handed over to the script instead of reading them out of `LocalConfiguration.php`

**Comment by schrej-zz** [comment]
@Apen,
I rewrote the according 2 files (SavewebsiteTask.php and save_typo3_website.sh.
The PHP now reads the DB values out of the $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default'] and hands them over to the shell script.
In the shell script, I replaced the read out of the LocalConfiguration.php by assigning the db variables by the handed over values.
Curently, the db parameters where handed over as single values. Maybe also the whole array can be handed over, but I never dealt with array in handig over array from PHP to bash.
Can I send you the files in a PM, or should I make a PULL REQUEST?

Joerg
PS: Solved also #19
